tools/osbuild-dev

```diff
@@ -148,13 +148,17 @@ class Manifest:
                 data = json.load(f)
         except FileNotFoundError:
             con.print(f"[bold][red]Could not open file {path!r}[/red][/bold]")
-            return 1
+            raise SystemExit(1)
```

Comment:
I'm wondering if there is any benefit in raising the exception instead of calling `sys.exit(1)`, which is commonly used in other tools and stages?

Reply:
There is not; it's personal preference as it skips an import (which is likely already imported).

Reply:
I would prefer consistency, but do not care too much... :wink:

Reply:
I've changed all calls to `SystemExit(n)` with `sys.exit(n)` :)